tools/bro-to-vast/bro-to-vast.cpp

```diff
@@ -57,14 +58,74 @@ class config : public broker::configuration {
   }
 };
 
-// Constructs a result event for Bro.
-broker::bro::Event make_bro_event(std::string id, broker::data x) {
+/// Converts VAST data to the corresponding Broker type.
+broker::data to_broker(const vast::data& data) {
+  return caf::visit(vast::detail::overload(
+    [](const auto& x) -> broker::data {
+      return x;
+    },
+    [](caf::none_t) -> broker::data {
+      return {};
+    },
+    [](const vast::pattern& x) -> broker::data {
+      return x.string();
+    },
+    [](const vast::address& x) -> broker::data {
+      auto bytes = reinterpret_cast<const uint32_t*>(x.data().data());
+      return broker::address{bytes, broker::address::family::ipv6,
+                             broker::address::byte_order::network};
```

Comment:
Why are byte order and family hardcoded?

Reply:
Because the raw bytes are always represented in network byte-order. When providing bytes in host-order, they are swapped during construction. Since we are accessing the raw bytes here, we know that they are in network byte-order.